--- a/02_configure_host.sh
+++ b/02_configure_host.sh
@@ -21,11 +21,13 @@ early_deploy_validation
 manage_libvirtd() {
   case ${DISTRO} in
       centos9|rhel9)
-          for i in qemu network nodedev nwfilter secret storage interface; do
+          for i in qemu interface network nodedev nwfilter secret storage; do
               sudo systemctl enable --now virt${i}d.socket
               sudo systemctl enable --now virt${i}d-ro.socket
               sudo systemctl enable --now virt${i}d-admin.socket
           done
+          sudo systemctl enable virtproxyd.service
+          sudo systemctl enable --now virtproxyd{,-ro,-admin}.socket
           ;;
       *)
           sudo systemctl restart libvirtd.service
